src/state/component-repository/components/actions.js

```diff
@@ -298,7 +298,7 @@ const procceedWithInstall = (component, body, resolve, dispatch, logProgress, lo
           const versionUnavailable = data && data.message;
           if (versionUnavailable) {
             dispatch(addErrors([data.message]));
-            dispatch(addToast(data.message, TOAST_ERROR));
+            dispatch(addToast(data.message || { id: 'componentRepository.components.genericError' }, TOAST_ERROR));
```

Comment:
I think we should save `{ id: 'componentRepository.components.genericError' }` inside a `const DEFAULT_ERROR_MESSAGE` for centralize this value and don't replicate it.

Reply:
done!

Reply:
done!